v2/emailpassword/quick-setup/frontend.mdx

```diff
@@ -138,10 +138,14 @@ SuperTokens.init({
 <AppInfoForm askForAPIDomain>
 
 ```tsx
-import supertokens from "supertokens-website-script";
+import supertokens from "supertokens-web-js-script";
```

Comment:
Need to also change `<TabItem value="npm">` one

Reply:
Done in separate PR